### ISSUE TITLE
SKR Pro 1.1 HID Bootloader

### ISSUE THREAD

**Issue by schnello** [KevinOConnor/klipper#2949]
Hello.

Can anybody confirm that the new hid Bootloader for the SKR Pro works? Anytime if i flash the bootloader (https://github.com/Arksine/STM32_HID_Bootloader/releases/download/v0.6-beta/hid_bootloader_SKR_PRO.bin) with ST Link any further USB connection is not possible."lsusb" shows no USB device.

**Comment by klipper-gitissuebot** [comment]
Hi @schnello,

It did not look like there was a Klipper log file attached to this ticket. The log file has been engineered to answer common questions the Klipper developers have about the software and its environment (software version, hardware type, configuration, event timing, and hundreds of other questions).

Unfortunately, too many people have opened tickets without providing the log. That consumes developer time; time that would be better spent enhancing the software. If this ticket references an event that has occurred while running the software then the Klipper log must be attached to this ticket. Otherwise, this ticket will be automatically closed in a few days.

For information on obtaining the Klipper log file see: https://github.com/KevinOConnor/klipper/blob/master/docs/Contact.md

The log can still be attached to this ticket - just add a comment and attach the log to that comment.

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being.

**Comment by Arksine** [comment]
It works.   I did upload a new version built with an older toolchain (no code changes) as it seems that there were some mixed results with the version built against gcc 8.  I also enabled issue reporting on that repo so further issues can be reported there.

One thing to keep in mind is that after flashing the bootloader you must manually enter it by setting BOOT1 high with a jumper.  After Klipper is flashed this is no longer necessary as "make flash" will enter the bootloader automatically and then upload Klipper.
